Python/compile.c

```diff
@@ -851,38 +857,37 @@ merge_consts_recursive(PyObject *const_cache, PyObject *o)
 }
 
 static Py_ssize_t
-compiler_add_const(PyObject *const_cache, struct compiler_unit *u, PyObject *o)
+compiler_add_const(struct compiler *c, PyObject *o)
```

Comment:
This is another example where the change removes compiler_unit* from the function arguments.